### Message type for the slider bar in my VIO project

# Header contains time stamp, sequence ID and frame ID
Header header
# Data includes:
float64 current_position    # current position of the slider endpoint
float64 set_position        # set position into the slider controller